app/js/lib/foxbox/recipes.js

```diff
@@ -62,22 +63,30 @@ export default class Recipe {
       `${this[p.net].origin}/api/v${this[p.settings].apiVersion}/services`)
       // Get all services but the recipes.
       .then(services => services.filter(service =>
-        service.adapter != 'thinkerbell-adapter' &&
-        service.adapter != 'webpush@link.mozilla.org'
+        service.adapter != 'thinkerbell-adapter'
+        && service.adapter != 'webpush@link.mozilla.org'
+        && service.adapter != 'console@link.mozilla.org'
+
+          // Deactivating IP camera from services with getters
+        && service.adapter != 'ip-camera@link.mozilla.org'
       ))
       // Keep only the services with getters.
       .then(services => services.filter(service =>
         Object.keys(service.getters).length
       ))
       // User friendly name.
       .then(services => services.map(service => {
-          switch (service.id) {
-            case 'service:clock@link.mozilla.org':
+          switch (service.adapter) {
+            case 'clock@link.mozilla.org':
               service.name = 'Everyday';
               break;
 
+            case 'ip-camera@link.mozilla.org':
```

Comment:
__question:__ mm I'm a bit puzzled, it looks like we've just filtered out ip camera service above?